app/index.js

```diff
@@ -9,98 +9,60 @@ class BlockstackGenerator extends Generator {
   constructor(args, opts) {
     // Calling the super constructor is important so our generator is correctly set up
     super(args, opts);
+    this.option('react', {
+      description: 'Generate a React app without prompting',
```

Comment:
`Generate a React project template.`
`Generate a plain Javascript project template.`
`Generate a Vue project template.`

Just omit "without prompting."

Reply:
Fixed! (Left out the trailing `.` to be consistent with the other built in help message descriptions)